syntax = "proto3";

package sift.common.type.v1;

import "google/api/field_behavior.proto";

option go_package = "azimuth/gen/protos/go/sift/common/type/v1;typev1pb";

message ChannelEnumType {
  string name = 1 [(google.api.field_behavior) = REQUIRED];
  uint32 key = 2 [(google.api.field_behavior) = REQUIRED];
}
